FLEDGE.md

```diff
@@ -761,6 +761,12 @@ even if multiple bid support is otherwise on, so they will have
 
 `generateBid()` has access to the `setPrioritySignalsOverride(key, value)` method. This adds an entry to the current interest group's `prioritySignalsOverrides` dictionary with the specified `key` and `value`, overwriting the previous value, if there was already an entry with `key`. If `value` is null, the entry with the specified key is deleted, if it exists.
 
+`generateBid()` has access to the `setBid(bids)` method. That takes the
```

Comment:
I vaguely recall some issue with Chrome permitting a single call to `setBid()`? Please note any limitatuion/caveat.